guides/decoding-blocks-transactions/getBlockById.js

```diff
@@ -20,13 +20,21 @@ getClient().then((client) => {
 	client.invoke("app:getBlockByID", {
 		id: blockId
 	}).then(res => {
-		console.log(res);
-		const blockObject = client.block.decode(res);
-		const blockJSON = client.block.toJSON(blockObject);
-		console.log(blockJSON);
+		const decodedBlock = client.block.decode(res);
+		const blockJSON = client.block.toJSON(decodedBlock);
+		const blockBuffer = client.block.fromJSON(blockJSON);
```

Comment:
i am not sure its buffer? its object? because data can be either JSON or object?

Reply:
yes you are right, updated it now!